warmshowers/Warmshowers.org
# Issue #3818: Security Update - Services

Redmine [#3818](https://pm.savaslabs.com/issues/3818)

**NOTE: this branch is based on** `feature/7-56-update`!!!

Hi @chrisarusso -

This PR applies the "Services" module security update to version 7.x-3.20.  (release [notes](https://www.drupal.org/node/2890353)).

### What I did
- Shelled into staging as user `www-data`
- Created a new branch off of `feature/7-56-update` b/c that branch has the most recent commit and is ahead of `develop`
- Ran `drush ups` and verified the "services" was the only security update available
- Ran `drush up services`
- Logged into the staging site as Admin and verified module was updated to the correct version, then clicked around and checked basic functionality.
- Committed the changes
- Pushed this PR for review

### To Test
- Please shell into the staging server
- check out this branch
- `drush uli` to login
- Verify that the "services" module is at version 7.x-3.20
- Check basic functionality

If this looks good, can we plan some time to chat about deployments? Since `develop` is behind I don't understand how the 7.56 core update got applied in production!

Let me know if you have any questions!
Ro
